resources/packs/gcp/gcp.lr

```diff
@@ -17,17 +17,61 @@ alias gcp.bigquery = gcp.project.bigqueryService
 alias gcp.storage = gcp.project.storageService
 
 // GCP Cloud Organization
-gcp.organization @defaults("id") {
+gcp.organization @defaults("name") {
   // Organization ID
-  id string
+  id() string
   // Organization name
-  name string
+  name() string
   // Organization state
-  lifecycleState string
+  state() string
+  // Deprecated. Use `state` instead.
+  lifecycleState() string
   // Organization IAM policy
   iamPolicy() []gcp.resourcemanager.binding
   // Access approval settings
   accessApprovalSettings() gcp.accessApprovalSettings
+  // List of folders
+  folders() gcp.folders
+  // List of projects
+  projects() gcp.projects
+}
+
+// GCP Folders
+private gcp.folders {
+  []gcp.folder
+  // Parent ID
+  parentId string
+  // Flat list of all folders within the organization
+  all() []gcp.folder
+}
+
+// GCP Folder
+private gcp.folder @defaults("name") {
+  // Folder ID
+  id string
+  // Folder name
+  name string
+  // Creation timestamp
+  created time
+  // Update timestamp
+  updated time
+  // Parent ID
+  parent string
```

Comment:
`parent` indicates a return of the parent resource. We should call this `parentId`. Currently the field name is also inconsistent with gcp.projects

Reply:
fixed